dts/bindings/interrupt-controller/arm,vim.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: ARMv7-R Vectored Interrupt Manager
+
```

Comment:
I think this naming scheme in this file introduces confusion, unlike NVIC, GIC .etc , VIM is not a standard ARM interrupt controller, but a TI specific interrupt controller.